README.md

```diff
@@ -19,3 +19,8 @@ let conjugatedWord = koreanConjugator.conjugate(‘하다’, {
 `npm run test`
 
 [Full API Documentation](https://github.com/llipio/conjugator/wiki)
+
+# Before commit a pull request (Must)
+```sh
+$ npm run eslint
+```
```

Comment:
since I don't know you have deployment system or not, I add this block for developers. If you have it, then I think you can hook pull request with eslint test on the deployment box.